src/main/docker/Dockerfile

```diff
@@ -34,6 +34,11 @@ RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | \
     tee /etc/apt/sources.list.d/yarn.list && \
     apt update && apt install yarn
 
+#
+# Install Cypress Dependencies (https://docs.cypress.io/guides/guides/continuous-integration.html#Dependencies)
+#
+RUN apt-get -y install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
```

Comment:
Wold you mind changing this to "apt install" instead of "apt-get -y install"
I'm no expert but it keeps it the same as how yarn is installed and this article suggests apt is a better way to go in general.

https://itsfoss.com/apt-vs-apt-get-difference/

Reply:
Sure! Thanks for the suggestion!